api/kyverno/v1/generaterequest_types.go

```diff
@@ -18,6 +17,9 @@ import (
 // +kubebuilder:printcolumn:name="status",type="string",JSONPath=".status.state"
 // +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
 // +kubebuilder:resource:shortName=gr
+// +kubebuilder:storageversion
+
+// GenerateRequest is a request to process generate rule.
```

Comment:
Should we remove GenerateRequest? It has been replaced by UpdateRequest.

Reply:
can you please address mw to pr which has this chnage?

Reply:
There are multiple PRs sent for this, https://github.com/kyverno/kyverno/pull/3601.

Let's log a separate issue (we can mark it as the "good first issue") and track the removal.

Reply:
sure